featurebyte/query_graph/node/cleaning_operation.py

```diff
@@ -225,6 +293,22 @@ class UnexpectedValueImputation(BaseCleaningOperation):
         description="List of values that are expected to be present."
     )
 
+    supported_dtypes: ClassVar[Optional[Set[DBVarType]]] = {
+        DBVarType.INT,
+        DBVarType.FLOAT,
+        DBVarType.DATE,
+        DBVarType.TIME,
+        DBVarType.TIMESTAMP,
+        DBVarType.TIMESTAMP_TZ,
+        DBVarType.BOOL,
+        DBVarType.CHAR,
+        DBVarType.VARCHAR,
+    }
```

Comment:
This set of dtypes seem common and also used in `DisguisedValueImputation`. Would it make sense to define it once and reuse?